Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "languageclient"
-version = "0.1.156"
```

Comment:
Changes here is not needed.

They will be automatically updated with other places as well when cut new release.

Reply:
Should I drop the change on install.sh as well?